src/Form/Layout.php

```diff
@@ -117,7 +117,7 @@ public function addGroup($label = null)
      *
      * @return static
      */
-    public function addSubLayout($seed = self::class, $addDivider = true)
+    public function addSubLayout($seed = [self::class], $addDivider = true)
```

Comment:
this was missed during merge

Reply:
Strange because i manully corrected it and force pushed.
Still tying to sort out the Table\Column refactor after the strict array seed merge...
We should not overlap with this as it causes a lot of time wasted.

Reply:
![image](https://user-images.githubusercontent.com/2228672/85802355-161f5500-b745-11ea-9064-37b147e41b20.png)

I now checked the commit history and this PR was failing before merge...

`Table\Column`, yes, originally, you started on develop:
![image](https://user-images.githubusercontent.com/2228672/85802476-5088f200-b745-11ea-95a8-0bf406df97c9.png)

so it will be probably better to simply merge `develop` there, as strict seeds PR is easy and resolving conflicts is basically adding `[` and `]` around based on that diff
- if you do not have the orignal branch based on `develop` - I pushed it under `refactor_tablecolumn_develop_based`

Reply:
One failing test on Column\FilterPopup remaining.
Problem is these files were completely moved so merging will not help. It is anyway based on latest develop / form-namespace.

Reply:
I fixed this PR. Originally, there was a bad merge and fix that was waiting here was not/still is not applied to develop, to merged it manually here.

How much RTM is this PR?

Reply:
After one or two more reviews we can merge. Also release schedule to be considered - if it will be included in 2.2 or 2.3

Reply:
We merge very quickly to prevent conflicts. Releases are up to Romains, in theory, you can pick any datr, take latest commit from each repo and do a release on your own if needed...

One more thing - in this PR, we normalize DropDown to Dropdown for Form\FieldDropdown. Immediately after this PR, we should fix all other names in whole ui repo.